ara/templates/playbook.html

```diff
@@ -66,21 +67,33 @@
   <tbody>
     {% for play in plays %}
     <tr>
-      <td>{{ playbook.time_start |datefmt }}</td>
-      <td>{{ playbook.time_end |datefmt }}</td>
-      <td>{{ playbook.duration |timefmt }}</td>
+      <td>{{ play.name }}</td>
+      <td>{{ play.time_start |datefmt }}</td>
+      <td>{{ play.time_end |datefmt }}</td>
+      <td>{{ play.duration |timefmt }}</td>
       <td>{{ macros.make_link('play.show_play', 'details', play=play.id) }}</td>
     </tr>
     {% endfor %}
   </tbody>
 </table>
 
 <h2>Tasks</h2>
+{% if not file_ %}
+<p>This is a list of all tasks encounted in the Ansible run.  You may
+limit this to a list of tasks contained in a particular file by going
+to {{ macros.make_link('playbook.show_playbook_files', 'the list of files',
+playbook=playbook.id) }}.</p>
+{% else %}
+<p>This is a list of tasks in <code>{{ file_.path }}</code>.  You may
+also see {{ macros.make_link('playbook.show_playbook',
+'the complete list of tasks', playbook=playbook.id) }}.</p>
```

Comment:
Nit but there's a space before the period in the UI

![Alt](http://i.imgur.com/w0HnUxN.png)